docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -435,7 +435,11 @@ described with the following additional property:
     See [AWS Glue Skip
     Archive](https://iceberg.apache.org/docs/latest/aws/#skip-archive).
   - `true`
-:::
+* - `iceberg.glue.cache-table-metadata`
+  - While updating the table in AWS Glue, store the table metadata with the
+    purpose of accelerating `information_schema.columns` queries.
```

Comment:
It improves listing speed on `system.metadata.table_comments` table.

Reply:
whats the downside of having this to `true` ..  and what is the advice for users. Also .. if we have more info like @ebyhr suggests on what specifically is faster that might help users decide. Unless we suggest they dont change it anyway.. but then we should say that as well

Reply:
The downside is that there is extra information stored in AWS Glue table parameters. Same as always in computing, "trade space for time" :)